python-lib/dku_deeplearning_image/tensorboard_handler.py

```diff
@@ -78,4 +80,31 @@ def tensorboard_wsgi_app(environ, start_response):
         start_response(status, headers)
         return [b"200"]
     else:
-        return __get_tb_app(__get_logs_path())(environ, start_response)
+        logs_path = __get_logs_path()
+        if logs_path:
+            return __get_tb_app(logs_path)(environ, start_response)
+        else:
+            status = '400'
+            headers = [('Content-type', 'text/html; charset=utf-8')]
+            start_response(status, headers)
+            return [get_no_folder_selected_error()]
+
+
+def get_no_folder_selected_error():
```

Comment:
Just a thought : 
Maybe it would be nice to have this html code in a separate/more generic function (inside an utility python file for example), that will be reusable in other context : I assume this type of error message appears in a similar way for other plugins too. The generic function could take as argument the actual content of the message and you'll just call it in this python module. But again just a thought :)

Reply:
That is actually a good thought! Let's keep it for later